Src/Microsoft.Dynamic/Hosting/Shell/Remote/RemoteConsoleCommandLine.cs

```diff
@@ -45,7 +45,8 @@ public class RemoteConsoleCommandLine : CommandLine {
                 // All bets are off while accessing the remote runtime. So we catch all exceptions.
                 // However, in most cases, we only expect to see RemotingException here.
                 if (!(exceptionDuringHandling is RemotingException)) {
-                    Console.WriteLine(String.Format("({0} thrown while trying to display unhandled exception)", exceptionDuringHandling.GetType()), Style.Error);
```

Comment:
This one could have just removed the String.Format and used the WriteLine overload which takes params.